benchexec/tools/witnesslint.py

```diff
@@ -0,0 +1,21 @@
+import benchexec.result as result
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for the witness checker (witnesslint)
+    (https://github.com/sosy-lab/sv-witnesses)
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("witnesslint.py")
+
+    def name(self):
+        return "witnesslint"
+
+    def determine_result(self, run):
+        if run.exit_code.value == 0:
+            return result.RESULT_TRUE_PROP
+        else:
+            return result.RESULT_FALSE_PROP
```

Comment:
This always returns `RESULT_FALSE_PROP`, even in cases where the tool crashes etc. Is there a way to distinguish these cases, e.g., is there a specific return code for `false`? If yes, checking for it here would make the tool-info more useful.

Reply:
The exit code of the linter now reflects whether the given witness is faulty or whether an internal error occurred -> adjusted `determine_result` accordingly.